src/helper/cli/cli.cpp

```diff
@@ -0,0 +1,105 @@
+#include "cli.hpp"
+
+#include <core/global/globals.hpp>
+#include <fancy.hpp>
+
+#include <unordered_map>
+
+namespace Soundux
+{
+    namespace Objects
+    {
+
+        typedef std::function<void(int, const char **)> CommandFunction;
+
+        struct Command
+        {
+            std::string description, example;
+            CommandFunction execFunction;
+            Command(const std::string &cmdDescription, const std::string &cmdExample,
+                    const CommandFunction &cmdFunction)
+                : description(cmdDescription), example(cmdExample), execFunction(cmdFunction)
+            {
+            }
+        };
+
+        void playsoundCommand(int, const char **);
```

Comment:
If you define `commandMap` in the header you don't have to forward declare these two functions here